packages/react-nft-checkout/.eslintrc.js

```diff
@@ -0,0 +1,60 @@
+module.exports = {
```

Comment:
u can override config for this package in the root dir, instead of creating separated one